tests/test_StatusCodeSender.js

```diff
@@ -23,35 +23,29 @@ describe("A status code sender", function () {
 		});
 	});
 
-	it("gives a Bad Credentials error on a 401.", function () {
-		return expectedErrorForStatusCode(errors.BadCredentialsError, 401);
-	});
-
-	it("gives a Payment Required error on a 402.", function () {
-		return expectedErrorForStatusCode(errors.PaymentRequiredError, 402);
-	});
-
-	it("gives a Request Entity Too Large error on a 413.", function () {
-		return expectedErrorForStatusCode(errors.RequestEntityTooLargeError, 413);
-	});
-
-	it("gives a Bad Request error on a 400.", function () {
-		return expectedErrorForStatusCode(errors.BadRequestError, 400);
-	});
-
-	it("gives an Unprocessable Entity error on a 422.", function () {
-		return expectedErrorForStatusCode(errors.UnprocessableEntityError, 422);
-	});
+	it("gives a custom message for 400", function () {
+		const payload = {
+			errors: [
+				{message: "custom message"}
+			]
+		};
+		return expectedDefaultError(400, payload);
+	})
 
-	it("gives a Too Many Requests error on a 429.", function () {
-		return expectedErrorForStatusCode(errors.TooManyRequestsError, 429);
-	});
+	it("returns an error message if payload is undefined", function () {
+		const payload = {
```

Comment:
I think we need to update the `expectedDefaultError` method so it can expect a message separate from what is in the payload. This would help this test to read a bit better because as is we are constructing a payload to pass in to compare the message which means the payload isn't actually undefined.

- [ ] add message param to method so we can have an undefined payload and still expect a specific message